subset/alliedSwitch/pom.xml

```diff
@@ -0,0 +1,95 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redstone</groupId>
+  <artifactId>alliedswitch</artifactId>
+  <version>0.0.1</version>
+  
+  <packaging>jar</packaging>
+
+  <name>alliedswitch</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+      <scope>test</scope>
+    </dependency>
+    
+<!-- not required 

```

Comment:
I like dead sections they are prompts to my future self, but seemingly have no value to you.
Will keep on a "dead section branch"

Reply:
The typical adage is that code is read more by other people than the original author -- so your "future self" isn't the right audience.  They typically also do more harm that good because, as comments, they get out of date and aren't updated. 

if they are actually useful then the text should be descriptive... e.g., if they're "not required" then they shouldn't be there. If there's a legit use for them then say what it is!  "This dependency is here for debugging network connectivity issues" or something like that.

Reply:
corrected xml spacing